virtual_rainforest/main.py

```diff
@@ -347,8 +265,8 @@ def vr_run(
 
         # Append updated data to the continuous data file
         if config["core"]["data_output_options"]["save_continuous_data"]:
-            outfile_path = output_current_state(
-                data, models_cfd, config["core"]["data_output_options"], time_index
+            outfile_path = data.output_current_state(
+                variables_to_save, config["core"]["data_output_options"], time_index
```

Comment:
Out of scope for this PR, but I'm wondering whether `output_current_state` could just take an `Iterable` of variables, rather than requiring a `list`.

Reply:
Actually, scratch this. I see that you're reusing `variables_to_save`, so it probably does make sense for it to be a `list` anyway.